rest/snippets/Config/UserEntity.yml

```diff
@@ -190,7 +190,10 @@ UserEntity:
           description: >-
             The Unix timestamp that the user was marked for soft deletion.
             Note:  Soft deletion removes all information from 'name' and 'contact details'
-
+    user_flags:
+      type: array
```

Comment:
@ishworkh Perhaps we have to define it as an array of string.
![image](https://user-images.githubusercontent.com/29536745/76042809-bed85780-5f90-11ea-93eb-f976fee2282e.png)

Reply:
can u point me to some example like that. I checked, i only saw `array`